File: StaggeredWithAccessibility/app/src/main/java/accessiblestaggered/novoda/com/staggeredwithaccessibility/MainActivity.java
package accessiblestaggered.novoda.com.staggeredwithaccessibility;

import android.app.Activity;
import android.os.Bundle;

import com.etsy.android.grid.StaggeredGridView;

public class MainActivity extends Activity {

    private StaggeredGridView gridView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        gridView = (StaggeredGridView) findViewById(R.id.gridview);
        gridView.setAdapter(new DummyAdapter(getApplicationContext(), getLayoutInflater()));
    }
}

File: multiplatform-http/android/src/main/java/com/novoda/playground/multiplatform/MainActivity.kt
package com.novoda.playground.multiplatform

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.novoda.playground.multiplatform.common.Presenter
import com.novoda.playground.multiplatform.common.Recipe
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), Presenter.View {

    private val presenter = Presenter()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun onStart() {
        super.onStart()
        presenter.startPresenting(this)
    }

    override fun onStop() {
        presenter.stopPresenting()
        super.onStop()
    }

    override fun render(recipe: Recipe) {
        recipe_name.text = recipe.header.title
    }
}
